src/elements/content-uploader/ItemRemove.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @flow
+ * @file Item remove component
+ */
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PlainButton from '../../components/plain-button/PlainButton';
+import Tooltip from '../../components/tooltip';
+import IconClose from '../../icons/general/IconClose';
+import messages from '../common/messages';
+
+type Props = {
+    isFailed: boolean,
+    isUploading: boolean,
+    onClick: Function,
+};
+
+const ItemRemove = ({ isFailed, isUploading, onClick }: Props) => {
+    const resin = {};
+    let target = null;
+
+    if (isUploading) {
+        target = 'uploadcancel';
+    } else if (isFailed) {
+        target = 'remove-failed';
+    }
+
+    if (target) {
+        resin['data-resin-target'] = target;
+    }
+
+    return (
+        <div className="bcu-item-action">
+            <Tooltip position="top-left" text={<FormattedMessage {...messages.remove} />}>
+                <PlainButton onClick={onClick} type="button" {...resin}>
```

Comment:
I think we recently added logic around disabling this button if the item was in a certain status. Has that been lost?